src/app/view-project/view-project.component.ts

```diff
@@ -0,0 +1,69 @@
+import {Component, OnChanges, OnInit} from '@angular/core';
+import {ProjectService} from '../../services/project-service/project.service';
+import {Project} from '../interfaces/project';
+import {DialogService} from '../../services/dialog-service/dialog.service';
+import {Observable} from 'rxjs';
+import {ResourceService} from '../../services/resource-service/resource.service';
+import {Resource} from '../interfaces/resource';
+
+
+@Component({
+  selector: 'app-projectviews',
+  templateUrl: './view-project.component.html',
+  styleUrls: ['./view-project.component.css']
+})
+export class ViewProjectComponent implements OnInit {
+  public projects: Observable<Project[]>;
+  public editResource: string;
+  constructor(private projectService: ProjectService, private resourceService: ResourceService, public dialog: DialogService) { }
+
+  ngOnInit(): void {
+    this.projects = this.projectService.getProjects();
```

Comment:
Observable variable names are usually marked with $ inside.
```suggestion
    this.projects$ = this.projectService.getProjects();
```